### ISSUE TITLE
Filter e-mails by date Not working

### ISSUE THREAD

**Issue by kvachakhia** [dacastro4/laravel-gmail#164]
Hello I want Filter Emails by date but it's not working
` $messages = \LaravelGmail::message()->subject('New customer message on September')->after('2020-09-20')->preload()->all();
`
Please Help and how I can get all message with the subject  "New customer message on" Because Month name is variable

**Comment by kvachakhia** [comment]
@dacastro4

**Comment by dacastro4** [comment]
If you try the same thing in your Gmail account, Are you getting what you're expecting?

**Comment by kvachakhia** [comment]
> If you try the same thing in your Gmail account, Are you getting what you're expecting?

Yes. I have one problem to I login in Gmail but after 3 days it's logout why?
